samples/bootloader/README.rst

```diff
@@ -218,12 +218,18 @@ Firmware versions using the |NSIB| are kept in the form of a *monotonic counter*
 Counter values are kept as slots in the flash memory, with each new counter value occupying a new slot.
 See :kconfig:option:`CONFIG_SB_MONOTONIC_COUNTER` for more details.
 
-To set the counter value of an application, build it using the :kconfig:option:`CONFIG_FW_INFO_FIRMWARE_VERSION` option:
+You can set the counter value for different elements in the bootloader sequence.
+For example, if your sample has both nRF Secure Immutable Bootloader (``b0``) and MCUboot (``b1``), you need to change the configuration value for the MCUboot by using :kconfig:option:`CONFIG_FW_INFO_FIRMWARE_VERSION` option.
```

Comment:
Hi, Is the process you are highlighting here the one mentioned in this page?
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_multi_image.html#ug-multi-image-variables

"You can make a project pass Kconfig configuration files, fragments, and devicetree overlays to child images by placing them in the child_image folder in the application source directory. The listing below describes how to leverage this functionality, where ACI_NAME is the name of the child image to which the configuration will be applied."

Reply:
@FrancescoSer Yes, that is the same process. However, I find the beginning of that section confusing because they're mixing defines and kconfig, and using `cmake` instead of `west`. So if linking here, I think we should state explicitly that it's the part about the `child_image` folder that is relevant.